packages/config/src/layer2s/types/Layer2TransactionApiV2.ts

```diff
@@ -18,7 +18,7 @@ export interface StarknetTransactionApiV2 {
 }
 
 export interface AztecTransactionApiV2 {
-  type: 'aztec'
+  type: 'aztec' | 'aztecconnect'
```

Comment:
I think that for any other system if they had a different api it would have its own distinct function. The only reason they are together right now is the fact that both systems are created by the same company, so I would be for splitting the logic. Duplication should be fine in this case.

Reply:
Yeah, I realised that too actually, so they are split now.